app/models/peep.rb

```diff
@@ -0,0 +1,12 @@
+class Peep
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :content, String
+  property :time, String
+  property :peeper, String
+
+  has 1, :user, {:through=>DataMapper::Resource}
```

Comment:
Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)